class.photon.php

```diff
@@ -290,6 +290,23 @@ public static function filter_the_content( $content ) {
 
 						// Supplant the original source value with our Photon URL
 						$photon_url = esc_url( $photon_url );
+						$zoom_url_2 = add_query_arg( 'zoom', 2, $photon_url );
+						$zoom_url_3 = add_query_arg( 'zoom', 3, $photon_url );
+						$zoom_url_4 = add_query_arg( 'zoom', 4, $photon_url );
+						$new_tag = str_replace( $src_orig, $photon_url, $new_tag );
+
+						$srcset_array = array(
+						    '1x' => $photon_url,
+						    '2x' => $zoom_url_2,
+						    '3x' => $zoom_url_3,
+						    '4x' => $zoom_url_4,
+						);
+						$srcset = '';
+						foreach ( $srcset_array as $size_label => $url ) {
+						     $srcset .= ', ' . esc_url( $url ) . ' '.$size_label;
+						}
+						$srcset = '"'.substr($srcset, 1).'"';
```

Comment:
Appending the `"` here, rather than down below when `preg_replace()`-ing it in feels rather odd.